CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Prefix your items with `(Template)` if the change is about the template and not
 - Optimized the .NET workloads install process.
 - Fixed the iOS application icon size.
 - Added VM Disposal in Functional Tests.
+- Added Dependency Injection validation in the development environment.
```

Comment:
Your commit type doesn't match the documented version.

Reply:
How? It's a feature.

Reply:
You're adding under 3.9, but your commit bumps that to 3.10

Reply:
Right.